scripts/linux/test_config_files_karl_exception.karl

```diff
@@ -0,0 +1,5 @@
+// this will cause an exception
+bullcrap(test_result += 1);
```

Comment:
Think of a different name. Bullcrap is not going in our repo.

Reply:
So, the reason this throws an exception is because users requested that if a function is used that has not been defined, an exception should be thrown.

Reply:
That sounds funny "Bullcrap is not going in our repo." Did you mean a pun too by accident.? lol.